appengine/java_appengine.bzl

```diff
@@ -339,7 +344,7 @@ default_java_toolchain(
     visibility = ["//visibility:public"],
 )
 """
-        # Bazel < 5.0.0
+        # Bazel < 6.0.0
 
     else:
         build_file_content = """
```

Comment:
Just to make sure: does this work with Bazel 5.x?

Reply:
Yes it does.
```
hvd@hvd:~/code/bazelbuild/rules_appengine$ git status && export USE_BAZEL_VERSION=5.4.0 && bazelisk --version && bazelisk test //...
On branch fix_broken_tools_dep
Your branch is up to date with 'origin/fix_broken_tools_dep'.

nothing to commit, working tree clean
bazel 5.4.0
INFO: Analyzed 23 targets (68 packages loaded, 25532 targets configured).
INFO: Found 22 targets and 1 test target...
INFO: Elapsed time: 4.494s, Critical Path: 3.23s
INFO: 55 processes: 40 internal, 12 linux-sandbox, 3 worker.
INFO: Build completed successfully, 55 total actions
//test/java:check_war                                                    PASSED in 0.4s

Executed 1 out of 1 test: 1 test passes.
INFO: Build completed successfully, 55 total actions
```